lib/_http_incoming.js

```diff
@@ -33,6 +33,7 @@ const {
 } = primordials;
 
 const { Readable, finished } = require('stream');
+const { kDestroy, destroy } = require('internal/stream')
```

Comment:
We _probably_ want to expose this so users can implement this to their own streams?

Reply:
I’m not sure... what are you thinking?

Reply:
I think they're useful for userland to override (probably)

Reply:
Again. I don't see the use for it. This is just to fix legacy streams inside of Node. Userland should preferably implement `_destroy` properly.